pkg/runtimeoptions/v1/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package runtimeoptions_v1 //nolint
```

Comment:
Was this change related to the api changes?

I was about to comment "why" did we pick this `runtimeoptions_v1` (and require a `nolint`), but saw the package already existed.

Reply:
this is what's in `api.pb.go` already, so just reflecting that. If i try to change to something else, linter will complain like so

```
pkg/runtimeoptions/v1/api.pb.go:1: : found packages runtimeoptions_v1 (api.pb.go) and runtimeoptions (doc.go) in pkg/runtimeoptions/v1 (typecheck)
// Code generated by protoc-gen-gogo. DO NOT EDIT.
```